topo_processor/metadata/loaders/metadata_loader_tiff.py

```diff
@@ -14,8 +16,19 @@ class MetadataLoaderTiff(MetadataLoader):
     def is_applicable(self, item: Item) -> bool:
         return is_tiff(item.path)
 
-    def add_metadata(self, item: Item) -> None:
-        # TODO use GDAL to load metadata from the tiff file if it exists
+    async def add_metadata(self, item: Item) -> None:
+        with rasterio.open(item.path) as tiff:
+            # projection
+            if tiff.crs:
+                if "projection" not in item.stac_item.stac_extensions:
+                    item.stac_item.stac_extensions.append("projection")
+                item.stac_item.properties.update({"proj:epsg": epsg_code(tiff.profile["crs"])})
+            else:
+                item.stac_item.properties.update({"proj:epsg": None})
```

Comment:
you will still need projection extension if you do this.

Is setting a None here needed? or is it better to just not add the data.

Reply:
I think setting it as None makes it more explicit to the users. So that they know we didn't forget to add it but it is really None